packages/react-vega/src/VegaEmbed.tsx

```diff
@@ -16,7 +16,7 @@ export type VegaEmbedProps = {
   signalListeners?: SignalListeners;
   style?: CSSProperties;
   onNewView?: ViewListener;
-  onError?: (error: Error) => void;
+  onError?: (error: Error, containerRef?: HTMLDivElement) => void;
```

Comment:
If the containerRef is always passed to the listener then don't add `?`.

```suggestion
  onError?: (error: Error, containerRef: HTMLDivElement) => void;
```

Reply:
It's there so that we don't break TypeScript builds for downstream consumers who have not implemented it.

Reply:
https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters-in-callbacks

Reply:
I didn't know this nuance, thanks! Fixed.

Reply:
Thanks for adding the param :)